### PR TITLE
DISTMYSQL-271 Release notes for PDPS 8.0.31

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/release-notes-ps-v8.0.31.md
modified:   docs/release-notes.md
modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/release-notes-ps-v8.0.31.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    }
+    tr:nth-child(even){
+        background-color:#f5f5f5
+    }
+    tr:hover {
+        background-color: #dddd;
+    }
+</style>
+
+| Release date         | February , 2022 |
```
Comment (patrickbirch) [inline docs/release-notes-ps-v8.0.31.md]: February 2023 ?

File: docs/release-notes-ps-v8.0.31.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+# Percona Distribution for MySQL 8.0.31 using Percona Server for MySQL (2022-02-)
```
Comment (patrickbirch) [inline docs/release-notes-ps-v8.0.31.md]: 2023- ?

Comment (alina-derkach-oaza) [reply docs/release-notes-ps-v8.0.31.md]: Updated

File: docs/release-notes-ps-v8.0.31.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes-ps-v8.0.31.md]: updated

File: docs/release-notes-ps-v8.0.31.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    ```sql
+    SQL> UPDATE PERFORMANCE_SCHEMA.SETUP_INTRUMENTS SET FLAGS="controlled" 
+    WHERE NAME='memory/sql/NET::buff';
+    ```
+
+* The `audit_log_flush` variable has been deprecated and will be removed in future releases.
+
+Find the full list of bug fixes and changes in the [MySQL 8.0.31 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-31.html).
+
+Review each component’s release notes for What’s new, improvements, or bug fixes.
```
Comment (patrickbirch) [inline docs/release-notes-ps-v8.0.31.md]: Make this the first paragraph of the next header. 
Otherwise. Good.

**Review by patrickbirch — APPROVED** [review]
Good

**Review by eleo007 — APPROVED** [review]

File: docs/release-notes-ps-v8.0.31.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/release-notes-ps-v8.0.31.md]: Updated

**Pull request merged**